src/deadline_test_fixtures/deadline/worker.py

```diff
@@ -493,7 +545,19 @@ def _start_worker_agent(self) -> None:  # pragma: no cover
 
     @property
     def worker_id(self) -> str:
-        cmd_result = self.send_command("cat /var/lib/deadline/worker.json  | jq -r '.worker_id'")
+        if self.configuration.operating_system.name == "AL2023":
+            cmd_result = self.send_command(
+                "cat /var/lib/deadline/worker.json  | jq -r '.worker_id'"
+            )
+        else:
+            cmd_result = self.send_command(
+                " ; ".join(
+                    [
+                        "$worker=Select-String -Path C:\ProgramData\Amazon\Deadline\Cache\worker.json -Pattern 'worker_id' | Select-Object -ExpandProperty  Line | ConvertFrom-Json",
```

Comment:
I think this logic assumes the file is minified JSON. I think going with:

```suggestion
                        "$worker=Get-Content -Raw C:\ProgramData\Amazon\Deadline\Cache\worker.json | ConvertFrom-Json",
```

is more robust to any valid JSON.

Reply:
Updated and tested.